workbench/views.py

```diff
@@ -80,14 +79,20 @@ def show_scenario(request, scenario_id, view_name='student_view', template='work
     })
 
 
-def user_list(request):
+def user_list(_request):
```

Comment:
If we're not using the `request` parameter, why even include it at all? Why not remove it?